### PR TITLE
Readme Update - Dev

### PR DESCRIPTION
**Pull request by AustinCasteel**
## Description

Updated the readme to remove coveralls and fix the codacy badge to reflect what it truly is.

## Related Issue

[Issue for this PR](https://github.com/NaomiProject/Naomi/issues/133)
[Past related Issue](https://github.com/NaomiProject/Naomi/issues/91)

[Master Branch PR](https://github.com/NaomiProject/Naomi/pull/134)

## Motivation and Context

Trying to keep the readme updated and current

## How Has This Been Tested?

Not Relevant...

## Screenshots (if appropriate):
Not Relevant...

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,9 +72,8 @@ You can [download](https://projectnaomi.com/download) latest installable version
 
 This software uses code from the initial Jasper Project & the Jasper2fork.
 
-- [JasperProject](http://jasperproject.github.io/)
-- [Jasper2Fork](https://github.com/andweber/j2f)
```
Comment (TuxSeb) [inline README.md:80]: Note: Naomi project was previously named J2F (Jasper2fork), andweber's repository is just a fork of it.

Comment (AustinCasteel) [reply README.md:80]: Ok, that section did not change. Just fixing the markdown list style.

**Review by aaronchantrill — APPROVED** [review]
This all looks fine to me. Just removing a vestigial badge and changing some bullets - simple. Sorry this took so long, I wasn't really paying attention.

**Comment by AustinCasteel** [comment]
Keep it on hold for now, since py3 has been pulled I will change the python badge as well.

**Comment by AustinCasteel** [comment]
Ready for merge!

**Review by redragonx — APPROVED** [review]
Read the changes, good to go.

**Pull request merged**